src/shared/helpers/user-is.ts

```diff
@@ -0,0 +1,15 @@
+import { CommonUserSchema } from '@viaa/avo2-types/types/user';
+
+import { SpecialUserGroup } from '../../admin/user-groups/user-group.const';
+
+export function userIsPupil(user: CommonUserSchema) {
+	return userIs(user, SpecialUserGroup.Pupil);
```

Comment:
Will this one still be used? Shouldn't all pupils either be ElementaryPupil or SecondaryPupil after the migration?

Reply:
User group was renamed, not created again. `SpecialUserGroup.Pupil` will refer to secondary pupils post-migration.

Reply:
ok, so maybe we should also rename the name of this entry in the enum, but keep the value the same?
```SpecialUserGroup.PupilSecundary```?

Reply:
Ok, then we should probably rename the enum to PupilSecundary but leave its underlaying value the same